docs/reference/release.md

```diff
@@ -28,17 +68,23 @@ git push origin v3.1.1
 ```
 4. Create a merge PR against master that only bumps the version number in `gutenberg.php`, `package.json`, and `package-lock.json`.
 
-## Build the Release
+### Build the Release
 
 Note: The `1.x.0` notation `git` and `svn` commands should be replaced with the version number of the new release.
 
 * Run `git checkout master` and `git pull`. Make sure your local `master` is up to date; you can confirm this by opening `gutenberg.php` and checking for the version bump you merged previously.
 * Run `./bin/build-plugin-zip.sh` from the root of project. This packages a zip file with a release build of `gutenberg.zip`.
```

Comment:
We have a warning in this script saying you're not master, we should remove it because aside tagging the first RC, we won't use master to release the versions anymore.